sdks/python/apache_beam/runners/portability/job_server_test.py

```diff
@@ -35,13 +35,15 @@ def java_arguments(
         artifact_port,
         '--expansion-port',
         expansion_port
```

Comment:
Looks like a comma is missing here (and some other test code also doesn't compile)

Reply:
Thank you for your cooperation. I will update. :)